File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-extend-ranges-in-documents_6.vb
        ' Define a range of 7 characters.
        Dim rng As Word.Range = Me.Application.ActiveDocument.Range(Start:=0, End:=7)

        ' Move the start position 7 characters.
        rng.MoveStart(Unit:=Word.WdUnits.wdCharacter, Count:=7)

        ' Move the end position 7 characters.
        rng.MoveEnd(Unit:=Word.WdUnits.wdCharacter, Count:=7)

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-extend-ranges-in-documents_6.cs
            // Define a range of 7 characters.
            Word.Range rng = this.Application.ActiveDocument.Range(0, 7);

            // Move the start position 7 characters.
            rng.MoveStart(Word.WdUnits.wdCharacter, 7);

            // Move the end position 7 characters.
            rng.MoveEnd(Word.WdUnits.wdCharacter, 7);